src/QirRuntime/test/QIR-tracer/generate.py

```diff
@@ -0,0 +1,39 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+import sys, os, platform, subprocess, datetime, shutil
+
+# =============================================================================
+#  Generates QIR files for all *.qs files in this folder
+#  Accepts arguments:
+#    path to qsc.exe (absolute or rely on Path env)

```

Comment:
Is not applicble?
> or rely on Path env

Reply:
why not? if you add the folder where qsc.exe is located to %PATH%, it will be found, no?